hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc

```diff
@@ -8,13 +8,15 @@ toc::[]
 
 Before you get started with {hcp} for {product-title}, you must properly label nodes so that the pods of hosted clusters can be scheduled into infrastructure nodes. Node labeling is also important for the following reasons:
 
-* To ensure high availability and proper workload deployment. For example, you can set the `node-role.kubernetes.io/infra` label to avoid having the control-plane workload count toward your {product-title} subscription.
+* To ensure high availability and proper workload deployment. For example, you can set the `node-role.kubernetes.io/infra` label to avoid having the control plane workload count toward your {product-title} subscription.
```

Comment:
Would recommend that you put the "why" before the "what", so that it reads: "For example, to avoid..., set the ... label..."